.github/workflows/integtest.yaml

```diff
@@ -107,6 +132,24 @@ jobs:
           done
           echo "::error title=npm run integ-test failed::Failed after $INTEG_TEST_ATTEMPTS attempts"
           exit 1
+      - name: run python tests
+        if: steps.get_language.outputs.language == 'py'
+        working-directory: ${{ matrix.app_to_test }}
+        run: |
+          # Note: We shouldn't need a global retry, but we do, due to async sets. See issues #44 and #33.
```

Comment:
We could use our new, reusable retry action for this retry (though I realize this is consistent with the js equivalent, and I agree with this decision — if we wanted to use the other action, we should do it for both).

https://github.com/klothoplatform/gh-action-retry

Reply:
If we had a readme for the action, I'd go for it. 😉

Reply:
touché!